modules/installation-configuration-parameters.adoc

```diff
@@ -1150,6 +1158,10 @@ at least two zones.
 |The name of the Azure cloud environment that is used to configure the Azure SDK with the appropriate Azure API endpoints. If empty, the default value `AzurePublicCloud` is used.
 |Any valid cloud environment, such as `AzurePublicCloud` or `AzureUSGovernmentCloud`.
 
+|`platform.azure.defaultMachinePlatform.vmNetworkingType`
+|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance.
+|`Accelerated` or `Basic`. If `controlPlane.platform.azure.type` and `compute.platform.azure.type` support `Accelerated` networking, installer by default enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Comment:
Actually, user can also set instance type for control plane and worker via `platform.azure.defaultMachinePlatform.type`.
How about rewording as below?
> if instance type of control plane and worker supports `Accelerated` networking, installer by default enables `Accelerated` networking, otherwise the default networking type is `Basic`.

I think same update also needs on `controlPlane.platform.azure.vmNetworkingType` and `compute.platform.azure.vmNetworkingType`